src/Sniff/ForbiddenSuperGlobalSniff.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace ShopSys\CodingStandards\Sniff;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+
+/**
+ * Inspired by
+ * @see https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/master/WordPress/Sniffs/VIP/SuperGlobalInputUsageSniff.php
+ */
+final class ForbiddenSuperGlobalSniff implements Sniff
+{
+    /**
+     * @var string[]
+     */
+    private $superglobalVariables = ['$_COOKIE', '$_GET', '$_FILES', '$_POST', '$_REQUEST', '$_SERVER'];
+
+    /**
+     * @return int[]
+     */
+    public function register()
+    {
+        return [T_VARIABLE];
+    }
+
+    /**
+     * @param int $position
```

Comment:
missing annotation for $file parameter

Reply:
The `File` type is actually in the argument already

Reply:
Ok, thanks for clarification.